tests/pytorch_tests/model_tests/feature_models/lut_quantizer_test.py

```diff
@@ -108,19 +109,22 @@ def create_feature_network(self, input_shape):
 
     def compare(self, quantized_models, float_model, input_x=None, quantization_info=None):
         quantized_model = quantized_models.get('lut_quantizer_test')
-        self.unit_test.assertTrue(np.sum(
-            np.abs(torch_tensor_to_numpy(quantized_model.conv1.weight) - torch_tensor_to_numpy(
-                quantized_model.conv2.weight))) > 0)
-        self.unit_test.assertFalse(np.sum(
-            np.abs(torch_tensor_to_numpy(quantized_model.conv2.weight) - torch_tensor_to_numpy(quantized_model.conv3.weight))) > 0)
+        q_weight1 = quantized_model.conv1.get_quantized_weights()['weight']
+        q_weight2 = quantized_model.conv2.get_quantized_weights()['weight']
+        self.unit_test.assertTrue(np.sum(np.abs(torch_tensor_to_numpy(q_weight1) -
+                                                torch_tensor_to_numpy(q_weight2))) > 0)
+        q_weight3 = quantized_model.conv3.get_quantized_weights()['weight']
+        self.unit_test.assertFalse(np.sum(np.abs(torch_tensor_to_numpy(q_weight3) -
+                                                 torch_tensor_to_numpy(q_weight3))) > 0)
 
 
 class LUTActivationQuantizerTest(BasePytorchTest):
     """
     This test checks that activation are quantized correctly using LUT quantizer
     """
     def __init__(self, unit_test, activation_n_bits=4):
-        super().__init__(unit_test)
+        super().__init__(unit_test,
+                         experimental_exporter=False)  # remove experimental_exporter when activation lut quantizer is added to inferable quantizers
```

Comment:
Please add TODO to the comment so it'll be easier to track when it'll be supported